src/Strategy/ParamStrategy.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace League\Route\Strategy;
+
+use League\Route\Http\Exception as HttpException;
+use League\Route\Route;
+use RuntimeException;
+
+class ParamStrategy extends AbstractStrategy implements StrategyInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function dispatch(callable $controller, array $vars, Route $route = null)
+    {
+        if (method_exists($this->getContainer(), 'call')) {
```

Comment:
This should be converted to an `instanceof` check against an interface that defines `call`.